.scripts/common.ts

```diff
@@ -112,14 +112,31 @@ function isPackageFolderPath(folderPath: string, packagesToIgnore: string[]): bo
   return result;
 }
 
+function shouldProcessFolderPath(folderPath: string, folderNamesToIgnore: string[]): boolean {
+  if (!contains(folderNamesToIgnore, getName(folderPath))) {
```

Comment:
It looks harder to understand.
Would it help better if you add a comment above with an example?
Such as..
```
folderNamesToIgnore = [ "service-bus", "eventgrid", ... ]; // Coming from /sdk/service/<service-sdk-name> 
folderPath = "..../sdk/servicebus/service-bus"
getName(folderPath) = "service-bus" 
```

Reply:
Similarly, at other places too.